code/turf/turf.dm

```diff
@@ -898,6 +898,33 @@
 /turf/unsimulated/wall/solidcolor/black
 	icon_state = "black"
 
+///old title card turf
+/turf/unsimulated/wall/titlecard
+	appearance_flags = TILE_BOUND
+	fullbright = 1
+	icon = 'icons/misc/widescreen.dmi' //fullscreen.dmi
```

Comment:
Move the widescreen.dmi file to the `assets` folder into a relevant subdirectory and do something like `icon = file("assets/icons/widescreen.dmi")` in `New` instead of doing this. (To reduce rsc size.)

Reply:
I have made the requested adjustments